apps/src/lib/levelbuilder/MarkdownEnabledTextarea.jsx

```diff
@@ -91,27 +112,38 @@ export default class MarkdownEnabledTextarea extends React.Component {
             style={styles.input}
             value={this.props.markdown}
           />
-          {this.props.features.imageUpload && (
-            <button
-              className="btn"
-              onClick={this.handleOpenUploadImage}
-              type="button"
-            >
-              <i style={styles.icon} className="fa fa-plus-circle" />
-              Image
-            </button>
-          )}
-          {this.props.features.resourceLink && (
-            <button
-              className="btn"
-              onClick={this.handleOpenAddResourceLink}
-              type="button"
-            >
-              <i style={styles.icon} className="fa fa-plus-circle" />
-              Resource
-            </button>
-          )}
+          <div className="btn-toolbar">
+            <div className="btn-group">
+              <a
+                className="btn dropdown-toggle"
+                data-toggle="dropdown"
+                href="#"
```

Comment:
FYI @madelynkasula that we are using bootstrap + jquery for dropdown functionality here and a few other places. this is probably an interesting case to consider re: component library. Not necessarily blocking for this PR, but you may be best positioned to share out whether it is important for us to avoid bootstrap functionality in react code. My understanding was that react was preferred over bootstrap and/or jquery, but I'm not sure we're very strict about that or whether there are negative implications in this case.